build/build-utils.js

```diff
@@ -14,17 +14,31 @@ const terser = require('terser');
  * @param {string} file
  */
 function minifyFileTransform(file) {
+  if (!file.endsWith('.js')) {
+    return new stream.Transform({
+      transform(chunk, enc, next) {
+        this.push(chunk);
+        next();
+      },
+    });
+  }
+
+  let code = '';
   return new stream.Transform({
     transform(chunk, enc, next) {
-      if (file.endsWith('.js')) {
-        const result = terser.minify(chunk.toString());
-        if (result.error) {
-          throw result.error;
-        }
+      code += chunk.toString();
+      next();
+    },
+    final(next) {
+      const result = terser.minify(code);
+      if (result.error) {
+        throw result.error;
+      }
 
+      if (result.code) {
+        // const saved = code.length - result.code.length;
```

Comment:
Figured some wouldn't want this logged all the time.

Reply:
Who sees this? maybe behind a debug flag or something instead?

the dead code option is probably my last preference among

- just log it anyway
- log it when asked with a flag/env var/something
- never log it
- commented code